docs/export_results.rst

```diff
@@ -68,7 +67,21 @@ To save images without creating a ``ti.GUI``, use ``ti.imwrite``. For example:
         ti.imwrite(pixels.to_numpy(), filename)
         print(f'The image has been saved to {filename}')
 
-Taichi offers helper functions that read, write, and show images. Please check out :ref:`gui` for more details.
+- ``ti.imwrite`` can export both Taichi tensors ``ti.Matrix``, ``ti.Vector`` or ``ti.var`` and numpy tensors ``np.ndarray``. 
+- Same as above ``ti.GUI.show(filename)``, the image format (``png``, ``jpg`` and ``bmp``) is also controlled by the suffix of ``filename`` in ``ti.imwrite(filename)``.
+- Meanwhile, The resulted image type (``grayscale``, ``RGB``, or ``RGBA``) is determined by **the number of channels in the input tensor**, aka the length of the third dimension, i.e. ``tensor.shape[2]``. 
+- For example, A tensor that has shape ``[w, h]`` or ``[w, h, 1]`` will be exported as a ``grayscale`` image.
+- If you want to export ``RGB`` or ``RGBA`` images instead, the input tensor should have a shape ``[w, h, 3]`` or ``[w, h, 4]`` respectively.
+
+.. note::
+    
+    A Taichi tensor can have a different datatype. Different datatype makes ``ti.imwrite`` have different behaviors.
+    
+    For a floating type(ti.f16, ti.f32) tensor, the pixel value on each channel of the image should be **a floating number between 0 and 1**.
+    
+    For an int type (ti.u8, ti.u16, etc) tensor, the values can be arbitrary valid **integer in its own bounds of datatype**.
```

Comment:
80－82 duplicate with gui.rst? Adding a link is good enough.

Reply:
Yes, I thought this question when I was writing indeed.
There has been an ref of gui.rst in L84. Will it be a little verbose if I ref it again in L82? How to deal with it properly?